ChaosMod/Effects/db/Peds/PedsSeriousKamikazes.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+	Effect by Moxi based on Killer Clowns by Last0xygen, explosives peds
+*/
+
+#include <stdafx.h>
+
+static std::list<Ped> kamikazeEnemies;
+static std::list<Object> bombObject;
+static int spawnTimer = -1;
+static Hash relationshipGroup;
+static int maxKamikazesToSpawn = 5;
+
+static Vector3 getRandomOffsetCoord(Vector3 startCoord, int minOffset, int maxOffset)
```

Comment:
Maybe use the function „ GetCoordAround“ to get random coordinates around the player with a random value for radius and angle :)

Reply:
Upper case naming?

Reply:
Done